dispatchers/Html.php

```diff
@@ -230,6 +263,30 @@ public function enqueue_assets() {
 		do_action( 'qm/output/enqueued-assets', $this );
 	}
 
+	/**
+	 * Determine whether the request is for an AMP page.
+	 *
+	 * @return bool
+	 */
+	private function is_amp_request() {
+		return function_exists( 'is_amp_endpoint' ) && is_amp_endpoint();
+	}
+
+	/**
+	 * Add AMP Dev Mode attributes to scripts and styles to omit them from AMP processing.
+	 */
+	private function filter_amp_dev_mode_element_xpaths() {
+		add_filter(
+			'amp_dev_mode_element_xpaths',
+			function ( $expressions ) {
+				$expressions[] = '//script[ @id = "query-monitor-js-extra" ]';
```

Comment:
Note: This expression will only work in WP≥5.5: https://github.com/WordPress/wordpress-develop/commit/d241ab3b55b06fed59e7ea03318e9f872399b390

I suppose we could do a version check and fall back to using `'//script[ contains( text(), "var qm_number_format =" ) ]'` if on WP<5.5.